BurnerB/FreeMentors-V2
# #168444648 Implement Lfa feedback

#### What does this PR do?
-Implements lfa feedback

#### Description of Task to be completed 
- drop table script
- remove isadmin and is mentor from response
- remove time from responses
- fix request session bug
- modify  jwt expiry

#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ch-lfa-feedback-db-168444648
* Run npm --init to install all dependancies.
* Run command `npm test` on your terminal
* Tests should pass
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#168444648](https://www.pivotaltracker.com/story/show/168444648)
#### Screenshots (if appropriate)
![remove password](https://user-images.githubusercontent.com/45785786/64756155-8c482780-d536-11e9-9dac-bfea65cacfed.png)
![testspassing](https://user-images.githubusercontent.com/45785786/64756158-8fdbae80-d536-11e9-9f4d-d16a0c7dd4fa.png)
